internal/controller/gateway_mutator.go

```diff
@@ -219,9 +217,9 @@ func (g *gatewayMutator) mutatePod(ctx context.Context, pod *corev1.Pod, gateway
 		Image:           g.extProcImage,
 		ImagePullPolicy: g.extProcImagePullPolicy,
 		Ports: []corev1.ContainerPort{
-			{Name: "aigw-metrics", ContainerPort: extProcMetricsPort},
+			{Name: "aigw-admin", ContainerPort: extProcAdminPort},
```

Comment:
so this could also be a breaking change. I think having two names pointing to the same port and removing it after the next release won't hurt. let me do that

Reply:
good idea thanks!